installation-usage.md

```diff
@@ -576,6 +577,116 @@ NAME                 STATUS      AGE
 my-recording-nginx   Installed   15s
 ```
 
+#### Merging per-container profile instances
+
+By default, each container instance will be recorded into a separate
+profile. This is mostly visible when recording pods managed by a replicating
+controller (Deployment, DaemonSet, etc.). A realistic example might
+be a workload being recorded in a test environment where the recorded
+Deployment consists of several replicas, only one of which is receiving
+the test traffic. After the recording is complete, only the container that
+was receiving the traffic would have container all the syscalls that were
+actually used.
+
+In this case, it might be useful to merge the per-container profiles
+into a single profile. This can be done by setting the `mergeStrategy`
+attribute to `containers` in the `ProfileRecording`. Note that the following
```

Comment:
Are `none` (the default) and `containers` the only two options for `mergeStrategy`?

Reply:
right now, yes. I was also wondering about an option called `workload` or even per-workload options like `deployment` like you suggested earlier which might produce just one policy per workload, but so far I haven't seen a good use-case for that.